### DISCUSSION TITLE
Build the qrcode  with a logo

### DISCUSSION THREAD

**Discussion by yaozhaoxu** [zxing-cpp/zxing-cpp#401] [Q&A] [answered]
hello. Does zxing-cpp have the interface building the qrcode that can set logo in the center of the qrcode.

**Comment by axxel** [comment, marked as answer]
No it does not. You can simply place a logo onto the code yourself. Just make sure the error correction level is high enough. You can check by scanning the code after you placed your logo on top of it.

**Comment by oehhar** [comment]
Placing a logo on a QR-Code is always out of spec and thus never supported, sorry for that.
As another hint, please take care to not cover any finder pattern, as they can not be corrected by error correction.
There are QR Code mode 2 sizes, which have a finder in the center, others do not have it.

Take care,
Harald
